cmd/minikube/cmd/delete.go

```diff
@@ -107,6 +122,30 @@ func runDelete(cmd *cobra.Command, args []string) {
 	if err := cmdcfg.Unset(pkg_config.MachineProfile); err != nil {
 		exit.WithError("unset minikube profile", err)
 	}
+
+	// Delete the .minikube folder if the flags are set
+	if viper.GetBool(purge) {
```

Comment:
If a user specifies `--purge`, I don't see why we should ask them again? 

My preference in behavior would be:

- If `--purge` is specified, and this is the only profile, just do what the user wanted.
- If `--purge` is specified, and there are more than one minikube profile, error out unless the user specifies `--force`.

Thoughts?

Reply:
This is fixed with the latest commit. And yes, I agree with what you mentioned! Now, if there is more than 1 profile, `--purge` will check for it and exit if `--all` flag is not specified. I decided to remove the `--force` flag. The combination of `--all --purge` should be enough. What do you think?